frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -69,7 +68,7 @@ export const DashboardHeaderContainer = styled.header<{
 }>`
   position: relative;
   z-index: 2;
-  background-color: var(--mb-color-bg-white);
+  background-color: var(--mb-color-bg-dashboard);
```

Comment:
I haven't verified all places, but I see in this PR uses all `--mb-color-bg-dashboard-card`, `--mb-color-bg-dashboard`, and `--mb-color-background`. There might be more. But I just want to point out that you're using different variables here. I hope they're all expected.